<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockerContainerScan</class>
 <widget class="QWidget" name="DockerContainerScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea_cresult">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>871</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>869</width>
      <height>179</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>221</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="0,6">
    <item>
     <widget class="QLabel" name="label_cssg">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SSG : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_cssg">
      <item>
       <property name="text">
        <string>NULL</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_cexit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>480</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cscan">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cinformation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Container Information</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>250</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="0,6">
    <item>
     <widget class="QLabel" name="label_cprofile">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Profile : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_cprofile">
      <item>
       <property name="text">
        <string>NULL</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_cresult">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Scan Result</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>191</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="0,0">
    <item>
     <widget class="QLabel" name="label_cmethod">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Method : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_cmethod">
      <item>
       <property name="text">
        <string>CVE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OVAL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XCCDF</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>281</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_24">
    <item>
     <widget class="QLabel" name="label_cid">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Container_ID : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_cid"/>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea_c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>1011</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1009</width>
      <height>179</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
